Makefile

```diff
@@ -312,7 +312,7 @@ catalog-push: ## Push a catalog image.
 
 .PHONY: signimage-build 
 signimage-build: ## Build docker image with the signer.
-	docker build -f Dockerfile.signimage -t $(SIGNER_IMG)
+	docker build -f Dockerfile.signimage -t $(SIGNER_IMG) .
```

Comment:
How does using `.` and also `-f Dockerfile.signimage` works?

Reply:
`.` is for the [build context](https://docs.docker.com/build/building/context/) and `-f` tells Docker which Dockerfile to use.